### DISCUSSION TITLE
Cannot disable breadcrumb on LoginComponent

### DISCUSSION THREAD

**Discussion by abhijithka** [udayvunnam/xng-breadcrumb#50] [Q&A] [answered]
Ideally, with the below router configuration, i would expect the breadcrumb to be gone on the login component. But the 'Home' breadcrumb is still displayed in that screen. I think this is a bug.

```
const routes: Routes = [
  {
    path: 'login', 
    component: LoginComponent,
    data: { breadcrumb: { skip: true, disable: true}}
  },
  {
    path: '',
    pathMatch: 'full',
    component: HomeComponent,
    data: { breadcrumb: 'Home' }
  }
]
```
Even with 'skip = true' in LoginComponent the breadcrumb 'Home' is still displayed

**Comment by abhijithka** [comment]
Are any developers of this library active? We are blocked by the above question for 12 days now and your comments in this will be a great help for us

**Comment by udayvunnam** [comment]
@abhijithka the current approach considers all routes within base path i.e ''.

For your use case, I would suggest keeping `<xng-breadcrumb>` within `HomeComponent` instead of `app.component.html`. Then it should be visible only within the scope of `HomeComponent` and skipped for `LoginComponent`

**Comment by abhijithka** [comment]
But our app structure is like

```
<header/>
<xng-breadcrumb/>
<router-outlet/>
<footer/>
```

And login component and home component all are loaded in place of router-outlet. If we place the breadcrumb is placed in HomeComponent won't it be available only in the child of HomeComponent ? We want it to be displayed for other components as well except LoginComponent without duplicating the code

**Comment by udayvunnam** [comment, marked as answer]
@abhijithka `breadcrumb: { skip: true}` hides the specific breadcrumb item but not the complete breadcrumb component.

Can you try ngIf on <xng-breadcrumb> and showing it on/off based on used loggedIn status
